CRM/Core/EntityTokens.php

```diff
@@ -63,7 +63,14 @@ public function evaluateToken(TokenRow $row, $entity, $field, $prefetch = NULL)
         \CRM_Utils_Money::format($fieldValue, $this->getCurrency($row)));
     }
     if ($this->isDateField($field)) {
-      return $row->format('text/plain')->tokens($entity, $field, \CRM_Utils_Date::customFormat($fieldValue));
+      try {
+        return $row->format('text/plain')
+          ->tokens($entity, $field, new DateTime($fieldValue));
+      }
+      catch (Exception $e) {
+        // Just catching this to stop another exception type bubbling up.
+        throw new CRM_Core_Exception('date handling failed ' . $e->getMessage());
```

Comment:
I'd be inclined to log an error and proceed with the old value?

Reply:
yeah ok